proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -13427,14 +13427,80 @@ function longFromRange( test )
   });
 
 };
+
 //
 
-function longFromRange1( test )
+function longFromRangeWithStep( test )
 {
+  test.case = 'equal numders of range';
+  var got = _.longFromRangeWithStep( [ 1, 1 ], 2 );
+  var expected = [ ];
+  test.identical( got, expected );
 
-  console.log('Hello world');
+  test.case = 'valid range with float step';
+  var got = _.longFromRangeWithStep( [ 0, 10 ], 1.5 );
+  var expected = [ 0, 1.5, 3, 4.5, 6, 7.5, 9 ];
+  test.identical( got, expected );
+
+  test.case = 'decrementing range with step < 0';
+  var got = _.longFromRangeWithStep( [ 10, 0 ], -1 );
+  var expected = [ 10, 9, 8, 7, 6, 5, 4, 3, 2, 1 ];
+  test.identical( got, expected );
+
+  test.case = 'step is undefined';
```

Comment:
Description of test case should describe difference of test case to each other. Also, description may contain common feature of test cases group.

The value of step argument is described. Please, describe the range argument in one style.